homeassistant/components/fixer/sensor.py

```diff
@@ -2,6 +2,7 @@
 from datetime import timedelta
 import logging
 
+from fixerio import Fixerio, exceptions
```

Comment:
Only `FixerioException` is used from `fixerio.exceptions`
Maybe it reads better when importing it separately

```suggestion
from fixerio import Fixerio
from fixerio.exceptions import FixerioException
```

(and adjust the single location where it is used).

Reply:
Great suggestion, updated the code accordingly in https://github.com/home-assistant/home-assistant/pull/29430/commits/dbe4acb04e0ae3abc93a37853701415a62191d79